src/cmdstan/command.hpp

```diff
@@ -594,6 +594,149 @@ int command(int argc, const char *argv[]) {
+    */
+    size_t u_params_vec_size = 0;
+    size_t u_params_size = 0;
+    size_t c_params_vec_size = 0;
+    size_t c_params_size = 0;
+
+    std::vector<double> u_params_r;
+    std::vector<double> c_params_r;
+    std::vector<size_t> dims_u_params_r;
+    std::vector<size_t> dims_c_params_r;
+    std::vector<int> dummy_params_i;
+
+    if (!(upars_file->is_default())) {
+      std::string u_fname(upars_file->value());
+      std::ifstream u_stream(u_fname.c_str());
+
+      std::shared_ptr<stan::io::var_context> upars_context
+          = get_var_context(u_fname);
+
+      u_params_r = (*upars_context).vals_r("params_r");
+      dims_u_params_r = (*upars_context).dims_r("params_r");
+
+      // Detect whether multiple sets of parameter values have been passed
+      // and set the sizes accordingly
+      u_params_vec_size = dims_u_params_r.size() == 2 ? dims_u_params_r[0] : 1;
+      u_params_size = dims_u_params_r.size() == 2 ? dims_u_params_r[1]
+                                                  : dims_u_params_r[0];
+    }
+
+    if (!(cpars_file->is_default())) {
+      std::string c_fname(cpars_file->value());
+      std::ifstream c_stream(c_fname.c_str());
+
+      std::shared_ptr<stan::io::var_context> cpars_context
+          = get_var_context(c_fname);
+
+      c_params_r = (*cpars_context).vals_r("params_r");
+      dims_c_params_r = (*cpars_context).dims_r("params_r");
+
+      c_params_vec_size = dims_c_params_r.size() == 2 ? dims_c_params_r[0] : 1;
+      c_params_size = dims_c_params_r.size() == 2 ? dims_c_params_r[1]
+                                                  : dims_c_params_r[0];
+    }
+
+    /*
+    Parameter names and dimensions required for constructing
+    array_var_context and subsequently transforming constrained parameters
+    to unconstrained scale
+    */
+    std::vector<std::string> param_names;
+    std::vector<std::vector<size_t>> param_dimss;
+    stan::services::get_model_parameters(model, param_names, param_dimss);
+
+    /*
+      Will store both constrained and unconstrained parameter sets in single
+      nested array so that only a single loop is required for calculation
+      and printing
+    */
+    size_t num_par_sets = c_params_vec_size + u_params_vec_size;
+    std::vector<std::vector<double>> params_r_ind(num_par_sets);
+
+    /*
+      An Eigen::Map with an inner stride is needed for viewing all parameter
+      values for a given set as a single vector when multiple parameter sets
+      are passed, as the var_context returns them appended by parameter, rather
+      than by set, i.e,:
+        [[a1, b1],[a2,b2]] => [a1, a2, b1, b2]
+    */
+    using StrideT = Eigen::Stride<1, Eigen::Dynamic>;
+    for (size_t i = 0; i < c_params_vec_size; i++) {
+      Eigen::Map<Eigen::VectorXd, 0, StrideT> map_r(
+          c_params_r.data() + i, c_params_size, StrideT(1, c_params_vec_size));
+
+      // Constrained parameters are unconstrained directly into the std::vector
```

Comment:
Remove this doc---this is just saying what `transform_inits` does.  If transform_init's doc isn't sufficient, add it there.